xbmc/interfaces/json-rpc/PlayerOperations.cpp

```diff
@@ -489,6 +492,150 @@ JSONRPC_STATUS CPlayerOperations::Zoom(const std::string &method, ITransportLaye
   }
 }
 
+std::map<std::string, ViewMode> viewModes =
```

Comment:
Thanks, good to have clear mapping of the relation between the two enum.
Could you add comment
// Matching pairs of  values from JSON type "Player.ViewMode" and C++ enum ViewMode
// Additions to enum ViewMode need to be added here and in the JSON type

Reply:
done